Tests/UnitTests/Networking/HTTPClientTests.swift

```diff
@@ -1857,19 +1857,32 @@ extension HTTPClientTests {
     }
 
     func testUsesServerProvidedBackoffIfPresent() {
-        let retryAfterMilliseconds = 100
+        let retryAfterSeconds = 100
 
-        let httpURLResponse = HTTPURLResponse(
+        var httpURLResponse = HTTPURLResponse(
             url: URL(string: "api.revenuecat.com")!,
             statusCode: HTTPStatusCode.tooManyRequests.rawValue,
             httpVersion: nil,
             headerFields: [
-                HTTPClient.ResponseHeader.retryAfter.rawValue: "\(retryAfterMilliseconds)"
+                HTTPClient.ResponseHeader.retryAfter.rawValue: "\(retryAfterSeconds)"
             ]
         )!
 
-        let backoffPeriod = self.client.calculateRetryBackoffTime(forResponse: httpURLResponse, retryCount: 2)
-        expect(backoffPeriod).to(equal(TimeInterval(0.1)))  // 0.1s == 100ms
+        var backoffPeriod = self.client.calculateRetryBackoffTime(forResponse: httpURLResponse, retryCount: 2)
```

Comment:
Since we're dealing with seconds now, I figured it'd be a good idea to make sure that we properly handle decimals as well as integers.